src/peep/peep.c

```diff
@@ -6915,6 +6915,7 @@ rct_map_element* get_banner_on_path(rct_map_element *path_element)
 
 static int banner_clear_path_edges(rct_map_element *mapElement, int edges)
 {
+	if (RCT2_GLOBAL(0x00F1AEDD, uint8) & 0x80) return edges;
```

Comment:
Agreed that this should change but since the staff path finding is not done yet this wont make any difference.